examples/webgl_materials_texture_3d.html

```diff
@@ -0,0 +1,213 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js webgl - 3D texture</title>
+		<meta charset="UTF-8">
+		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+		<style>
+			body {
+				background:#000;
+				padding:0;
+				margin:0;
+				overflow:hidden;
+			}
+
+			#info {
+				position: absolute;
+				top: 0px;
+				width: 100%;
+				color: #ffffff;
+				padding: 5px;
+				font-family:Monospace;
+				font-size:13px;
+				text-align:center;
+			}
+
+			#info a {
+				color: #ffffff;
+			}
+		</style>
+	</head>
+	<script id="vs" type="x-shader/x-vertex">
+	#version 300 es
+
+	uniform float depth;
+	uniform vec2 size;
+	out vec3 vUv;
+
+	void main() {
+
+		gl_Position = projectionMatrix * modelViewMatrix * vec4( position, 1.0 );
+
+		// Convert position.xy to 1.0-0.0
+
+		vUv.xy = position.xy / size + 0.5;
+		vUv.y = 1.0 - vUv.y; // original data is upside down
+
+		vUv.z = depth;
+
+	}
+	</script>
+
+	<script id="fs" type="x-shader/x-fragment">
+	#version 300 es
+
+	precision highp float;
+	precision highp int;
+	precision highp sampler3D;
+
+	uniform sampler3D diffuse;
+	in vec3 vUv;
+	out vec4 out_FragColor;
+
+	void main() {
+
+		vec4 color = texture( diffuse, vUv );
+		color.r *= 1.5; // lighten a bit
+		color.gb = vec2( color.r );
+
+		out_FragColor = vec4( color.rgb, 1.0 );
```

Comment:
nitpick: why not just `out_FragColor = vec4( color.r, color, r, color, r, 1.0 );` and omit the line above.

Reply:
Even better:

```glsl
out_FragColor = vec4( color.rrr, 1.0 );
```

Reply:
Updated, thanks.